.github/actions/azureml-test/action.yml

```diff
@@ -71,17 +71,17 @@ runs:
     - name: Setup python
```

Comment:
this is the content of the logs on azureml:
```
======Starting Image Build on Serverless Compute======
The run ID for the image build on serverless compute is imgbldrun_e4f6cd6
Additional logs for the run: https://ml.azure.com/experiments/id/prepare_image/runs/imgbldrun_e4f6cd6?wsid=/subscriptions/a30dea4f-623a-4bc5-808b-cbf9bc00f7a1/resourcegroups/recommenders_project_resources/workspaces/azureml-test-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47
2024-07-03T14:33:41: Logging into Docker registry: 978a92daa2ad4447aae1b21196dd4a9b.azurecr.io
2024-07-03T14:33:41: WARNING! Using --password via the CLI is insecure. Use --password-stdin.
2024-07-03T14:33:41: WARNING! Your password will be stored unencrypted in /root/.docker/config.json.
2024-07-03T14:33:41: Configure a credential helper to remove this warning. See
2024-07-03T14:33:41: https://docs.docker.com/engine/reference/commandline/login/#credentials-store

2024-07-03T14:33:41: Login Succeeded


2024-07-03T14:33:42: Running: ['docker', 'build', '-f', 'azureml-environment-setup/Dockerfile', '.', '-t', '978a92daa2ad4447aae1b21196dd4a9b.azurecr.io/azureml/azureml_fa83f0efe189b6dbcbb6f109441a392a', '-t', '978a92daa2ad4447aae1b21196dd4a9b.azurecr.io/azureml/azureml_fa83f0efe189b6dbcbb6f109441a392a:1']
2024-07-03T14:33:42: DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
2024-07-03T14:33:42:             Install the buildx component to build images with BuildKit:
2024-07-03T14:33:42:             https://docs.docker.com/go/buildx/

2024-07-03T14:33:42: Sending build context to Docker daemon  71.17kB
2024-07-03T14:33:42: Step 1/17 : FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu22.04@sha256:c237855f4f08218c4d130a356933b07592e33413bddabb697e98fc2f36b87332
2024-07-03T14:33:42: mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu22.04@sha256:c237855f4f08218c4d130a356933b07592e33413bddabb697e98fc2f36b87332: Pulling from azureml/openmpi4.1.0-ubuntu22.04
2024-07-03T14:33:42: 2ec76a50fe7c: Pulling fs layer
2024-07-03T14:33:42: 890e4054dcca: Pulling fs layer
2024-07-03T14:33:42: e1992d002114: Pulling fs layer
2024-07-03T14:33:42: 92f14880e04a: Pulling fs layer
2024-07-03T14:33:42: 0236928f6e79: Pulling fs layer
2024-07-03T14:33:42: 560ac843a698: Pulling fs layer
2024-07-03T14:33:42: 6a00999f16b3: Pulling fs layer
2024-07-03T14:33:42: 6ed2507ff55f: Pulling fs layer
2024-07-03T14:33:42: 91e332d30a0c: Pulling fs layer
2024-07-03T14:33:42: 513c77a0ff3d: Pulling fs layer
2024-07-03T14:33:42: 5c508012f890: Pulling fs layer
2024-07-03T14:33:42: 6a00999f16b3: Waiting
2024-07-03T14:33:42: 6ed2507ff55f: Waiting
2024-07-03T14:33:42: 91e332d30a0c: Waiting
2024-07-03T14:33:42: 513c77a0ff3d: Waiting
2024-07-03T14:33:42: 5c508012f890: Waiting
2024-07-03T14:33:42: 0236928f6e79: Waiting
2024-07-03T14:33:42: 92f14880e04a: Waiting
2024-07-03T14:33:42: 560ac843a698: Waiting
2024-07-03T14:33:42: e1992d002114: Verifying Checksum
2024-07-03T14:33:42: e1992d002114: Download complete
2024-07-03T14:33:42: 92f14880e04a: Verifying Checksum
2024-07-03T14:33:42: 92f14880e04a: Download complete
2024-07-03T14:33:42: 2ec76a50fe7c: Verifying Checksum
2024-07-03T14:33:42: 2ec76a50fe7c: Download complete
2024-07-03T14:33:42: 0236928f6e79: Download complete
2024-07-03T14:33:42: 560ac843a698: Verifying Checksum
2024-07-03T14:33:42: 560ac843a698: Download complete
2024-07-03T14:33:42: 6ed2507ff55f: Verifying Checksum
2024-07-03T14:33:42: 6ed2507ff55f: Download complete
2024-07-03T14:33:43: 890e4054dcca: Verifying Checksum
2024-07-03T14:33:43: 890e4054dcca: Download complete
2024-07-03T14:33:43: 91e332d30a0c: Verifying Checksum
2024-07-03T14:33:43: 91e332d30a0c: Download complete
2024-07-03T14:33:43: 5c508012f890: Verifying Checksum
2024-07-03T14:33:43: 5c508012f890: Download complete
2024-07-03T14:33:43: 513c77a0ff3d: Verifying Checksum
2024-07-03T14:33:43: 513c77a0ff3d: Download complete
2024-07-03T14:33:43: 6a00999f16b3: Verifying Checksum
2024-07-03T14:33:43: 6a00999f16b3: Download complete
2024-07-03T14:33:44: 2ec76a50fe7c: Pull complete
2024-07-03T14:33:51: 890e4054dcca: Pull complete
2024-07-03T14:33:51: e1992d002114: Pull complete
2024-07-03T14:33:51: 92f14880e04a: Pull complete
2024-07-03T14:33:53: 0236928f6e79: Pull complete
2024-07-03T14:33:53: 560ac843a698: Pull complete
2024-07-03T14:33:59: 6a00999f16b3: Pull complete
2024-07-03T14:33:59: 6ed2507ff55f: Pull complete
2024-07-03T14:34:00: 91e332d30a0c: Pull complete
2024-07-03T14:34:01: 513c77a0ff3d: Pull complete
2024-07-03T14:34:01: 5c508012f890: Pull complete
2024-07-03T14:34:01: Digest: sha256:c237855f4f08218c4d130a356933b07592e33413bddabb697e98fc2f36b87332
2024-07-03T14:34:01: Status: Downloaded newer image for mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu22.04@sha256:c237855f4f08218c4d130a356933b07592e33413bddabb697e98fc2f36b87332
2024-07-03T14:34:01:  ---> 239d28bc8ce2
2024-07-03T14:34:01: Step 2/17 : USER root
2024-07-03T14:34:01:  ---> Running in 17de8875196b
2024-07-03T14:34:02: Removing intermediate container 17de8875196b
2024-07-03T14:34:02:  ---> f1d37c34b0b6
2024-07-03T14:34:02: Step 3/17 : RUN mkdir -p $HOME/.cache
2024-07-03T14:34:02:  ---> Running in c5785c4a30d3
2024-07-03T14:34:04: Removing intermediate container c5785c4a30d3
2024-07-03T14:34:04:  ---> 8017ba9565b2
2024-07-03T14:34:04: Step 4/17 : WORKDIR /
2024-07-03T14:34:04:  ---> Running in 618c39279d29
2024-07-03T14:34:05: Removing intermediate container 618c39279d29
2024-07-03T14:34:05:  ---> eead7016f62e
2024-07-03T14:34:05: Step 5/17 : COPY azureml-environment-setup/99brokenproxy /etc/apt/apt.conf.d/
2024-07-03T14:34:06:  ---> b7671f93ad5f
2024-07-03T14:34:06: Step 6/17 : RUN if dpkg --compare-versions `conda --version | grep -oE '[^ ]+$'` lt 4.4.11; then conda install conda==4.4.11; fi
2024-07-03T14:34:06:  ---> Running in 89da7e7d0921
2024-07-03T14:34:09: Removing intermediate container 89da7e7d0921
2024-07-03T14:34:09:  ---> f551613be986
2024-07-03T14:34:09: Step 7/17 : COPY azureml-environment-setup/mutated_conda_dependencies.yml azureml-environment-setup/mutated_conda_dependencies.yml
2024-07-03T14:34:10:  ---> 88e5d3d7502b
2024-07-03T14:34:10: Step 8/17 : RUN ldconfig /usr/local/cuda/lib64/stubs && conda env create -p /azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc -f azureml-environment-setup/mutated_conda_dependencies.yml && rm -rf "$HOME/.cache/pip" && conda clean -aqy && CONDA_ROOT_DIR=$(conda info --root) && rm -rf "$CONDA_ROOT_DIR/pkgs" && find "$CONDA_ROOT_DIR" -type d -name __pycache__ -exec rm -rf {} + && ldconfig
2024-07-03T14:34:10:  ---> Running in 339500e7a8f0
2024-07-03T14:34:10: Retrieving notices: ...working... done
2024-07-03T14:34:10: Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
2024-07-03T14:34:11: Channels:
2024-07-03T14:34:11:  - anaconda
2024-07-03T14:34:11:  - conda-forge
2024-07-03T14:34:11:  - defaults
2024-07-03T14:34:11: Platform: linux-64
2024-07-03T14:34:27: Collecting package metadata (repodata.json): ...working... done
2024-07-03T14:34:28: Solving environment: ...working... done

2024-07-03T14:34:40: Downloading and Extracting Packages: ...working... done
2024-07-03T14:34:41: Preparing transaction: ...working... done
2024-07-03T14:34:43: Verifying transaction: ...working... done
2024-07-03T14:34:44: Executing transaction: ...working... done
2024-07-03T14:35:41: Installing pip dependencies: ...working... [91mPip subprocess error:
2024-07-03T14:35:41:   Running command git clone --filter=blob:none --quiet https://github.com/pymanopt/pymanopt.git /tmp/pip-req-build-rh5ss7zb
2024-07-03T14:35:41:   Running command git rev-parse -q --verify 'sha^1de3b6f47258820fdc072fceaeaa763b9fd263b0'
2024-07-03T14:35:41:   Running command git fetch -q https://github.com/pymanopt/pymanopt.git 1de3b6f47258820fdc072fceaeaa763b9fd263b0
2024-07-03T14:35:41:   Running command git clone --filter=blob:none --quiet https://github.com/recommenders-team/recommenders.git /tmp/pip-install-gp9yg5dt/recommenders_783800d8e7294cd7a841e525f4aed605
2024-07-03T14:35:41:   Running command git rev-parse -q --verify 'sha^842308827aa02d25c3863f2f6552351a94eb19d0'
2024-07-03T14:35:41:   Running command git fetch -q https://github.com/recommenders-team/recommenders.git 842308827aa02d25c3863f2f6552351a94eb19d0
2024-07-03T14:35:41:   Running command git checkout -q 842308827aa02d25c3863f2f6552351a94eb19d0
2024-07-03T14:35:41:   error: subprocess-exited-with-error
2024-07-03T14:35:41:   
2024-07-03T14:35:41:   × pip subprocess to install build dependencies did not run successfully.
2024-07-03T14:35:41:   │ exit code: 2
2024-07-03T14:35:41:   ╰─> [103 lines of output]
2024-07-03T14:35:41:       Ignoring numpy: markers 'python_version < "3.9"' don't match your environment
2024-07-03T14:35:41:       Collecting cython>=0.29
2024-07-03T14:35:41:         Using cached Cython-3.0.10-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (3.2 kB)
2024-07-03T14:35:41:       Collecting numpy==1.19.4
2024-07-03T14:35:41:         Downloading numpy-1.19.4.zip (7.3 MB)
2024-07-03T14:35:41:            ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 7.3/7.3 MB 58.6 MB/s eta 0:00:00
2024-07-03T14:35:41:         Installing build dependencies: started
2024-07-03T14:35:41:         Installing build dependencies: finished with status 'done'
2024-07-03T14:35:41:         Getting requirements to build wheel: started
2024-07-03T14:35:41:         Getting requirements to build wheel: finished with status 'done'
2024-07-03T14:35:41:       ERROR: Exception:
2024-07-03T14:35:41:       Traceback (most recent call last):
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/cli/base_command.py", line 180, in exc_logging_wrapper
2024-07-03T14:35:41:           status = run_func(*args)
2024-07-03T14:35:41:                    ^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/cli/req_command.py", line 245, in wrapper
2024-07-03T14:35:41:           return func(self, options, args)
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/commands/install.py", line 377, in run
2024-07-03T14:35:41:           requirement_set = resolver.resolve(
2024-07-03T14:35:41:                             ^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/resolver.py", line 95, in resolve
2024-07-03T14:35:41:           result = self._result = resolver.resolve(
2024-07-03T14:35:41:                                   ^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_vendor/resolvelib/resolvers.py", line 546, in resolve
2024-07-03T14:35:41:           state = resolution.resolve(requirements, max_rounds=max_rounds)
2024-07-03T14:35:41:                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_vendor/resolvelib/resolvers.py", line 397, in resolve
2024-07-03T14:35:41:           self._add_to_criteria(self.state.criteria, r, parent=None)
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_vendor/resolvelib/resolvers.py", line 173, in _add_to_criteria
2024-07-03T14:35:41:           if not criterion.candidates:
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_vendor/resolvelib/structs.py", line 156, in __bool__
2024-07-03T14:35:41:           return bool(self._sequence)
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/found_candidates.py", line 155, in __bool__
2024-07-03T14:35:41:           return any(self)
2024-07-03T14:35:41:                  ^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/found_candidates.py", line 143, in <genexpr>
2024-07-03T14:35:41:           return (c for c in iterator if id(c) not in self._incompatible_ids)
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/found_candidates.py", line 47, in _iter_built
2024-07-03T14:35:41:           candidate = func()
2024-07-03T14:35:41:                       ^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/factory.py", line 211, in _make_candidate_from_link
2024-07-03T14:35:41:           self._link_candidate_cache[link] = LinkCandidate(
2024-07-03T14:35:41:                                              ^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 293, in __init__
2024-07-03T14:35:41:           super().__init__(
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 156, in __init__
2024-07-03T14:35:41:           self.dist = self._prepare()
2024-07-03T14:35:41:                       ^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 225, in _prepare
2024-07-03T14:35:41:           dist = self._prepare_distribution()
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 304, in _prepare_distribution
2024-07-03T14:35:41:           return preparer.prepare_linked_requirement(self._ireq, parallel_builds=True)
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/operations/prepare.py", line 525, in prepare_linked_requirement
2024-07-03T14:35:41:           return self._prepare_linked_requirement(req, parallel_builds)
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/operations/prepare.py", line 640, in _prepare_linked_requirement
2024-07-03T14:35:41:           dist = _get_prepared_distribution(
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/operations/prepare.py", line 71, in _get_prepared_distribution
2024-07-03T14:35:41:           abstract_dist.prepare_distribution_metadata(
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/distributions/sdist.py", line 54, in prepare_distribution_metadata
2024-07-03T14:35:41:           self._install_build_reqs(finder)
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/distributions/sdist.py", line 124, in _install_build_reqs
2024-07-03T14:35:41:           build_reqs = self._get_build_requires_wheel()
2024-07-03T14:35:41:                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/distributions/sdist.py", line 101, in _get_build_requires_wheel
2024-07-03T14:35:41:           return backend.get_requires_for_build_wheel()
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_internal/utils/misc.py", line 751, in get_requires_for_build_wheel
2024-07-03T14:35:41:           return super().get_requires_for_build_wheel(config_settings=cs)
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_impl.py", line 166, in get_requires_for_build_wheel
2024-07-03T14:35:41:           return self._call_hook('get_requires_for_build_wheel', {
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_impl.py", line 321, in _call_hook
2024-07-03T14:35:41:           raise BackendUnavailable(data.get('traceback', ''))
2024-07-03T14:35:41:       pip._vendor.pyproject_hooks._impl.BackendUnavailable: Traceback (most recent call last):
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 77, in _build_backend
2024-07-03T14:35:41:           obj = import_module(mod_path)
2024-07-03T14:35:41:                 ^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/importlib/__init__.py", line 90, in import_module
2024-07-03T14:35:41:           return _bootstrap._gcd_import(name[level:], package, level)
2024-07-03T14:35:41:                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap>", line 1381, in _gcd_import
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap>", line 1354, in _find_and_load
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap>", line 1304, in _find_and_load_unlocked
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap>", line 1381, in _gcd_import
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap>", line 1354, in _find_and_load
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap>", line 1325, in _find_and_load_unlocked
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap>", line 929, in _load_unlocked
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap_external>", line 994, in exec_module
2024-07-03T14:35:41:         File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
2024-07-03T14:35:41:         File "/tmp/pip-build-env-qo_olf7o/overlay/lib/python3.12/site-packages/setuptools/__init__.py", line 18, in <module>
2024-07-03T14:35:41:           from setuptools.extern.six import PY3, string_types
2024-07-03T14:35:41:       ModuleNotFoundError: No module named 'setuptools.extern.six'
2024-07-03T14:35:41:       
2024-07-03T14:35:41:       [end of output]
2024-07-03T14:35:41:   
2024-07-03T14:35:41:   note: This error originates from a subprocess, and is likely not a problem with pip.
2024-07-03T14:35:41: error: subprocess-exited-with-error

2024-07-03T14:35:41: × pip subprocess to install build dependencies did not run successfully.
2024-07-03T14:35:41: │ exit code: 2
2024-07-03T14:35:41: ╰─> See above for output.

2024-07-03T14:35:41: note: This error originates from a subprocess, and is likely not a problem with pip.

2024-07-03T14:35:41: [0mRan pip subprocess with arguments:
2024-07-03T14:35:41: ['/azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/bin/python', '-m', 'pip', 'install', '-U', '-r', '/azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt', '--exists-action=b']
2024-07-03T14:35:41: Pip subprocess output:
2024-07-03T14:35:41: Collecting git+https://github.com/pymanopt/pymanopt.git@1de3b6f47258820fdc072fceaeaa763b9fd263b0 (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 2))
2024-07-03T14:35:41:   Cloning https://github.com/pymanopt/pymanopt.git (to revision 1de3b6f47258820fdc072fceaeaa763b9fd263b0) to /tmp/pip-req-build-rh5ss7zb
2024-07-03T14:35:41:   Resolved https://github.com/pymanopt/pymanopt.git to commit 1de3b6f47258820fdc072fceaeaa763b9fd263b0
2024-07-03T14:35:41:   Installing build dependencies: started
2024-07-03T14:35:41:   Installing build dependencies: finished with status 'done'
2024-07-03T14:35:41:   Getting requirements to build wheel: started
2024-07-03T14:35:41:   Getting requirements to build wheel: finished with status 'done'
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): started
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): finished with status 'done'
2024-07-03T14:35:41: Collecting recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0 (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Cloning https://github.com/recommenders-team/recommenders.git (to revision 842308827aa02d25c3863f2f6552351a94eb19d0) to /tmp/pip-install-gp9yg5dt/recommenders_783800d8e7294cd7a841e525f4aed605
2024-07-03T14:35:41:   Resolved https://github.com/recommenders-team/recommenders.git to commit 842308827aa02d25c3863f2f6552351a94eb19d0
2024-07-03T14:35:41:   Installing build dependencies: started
2024-07-03T14:35:41:   Installing build dependencies: finished with status 'done'
2024-07-03T14:35:41:   Getting requirements to build wheel: started
2024-07-03T14:35:41:   Getting requirements to build wheel: finished with status 'done'
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): started
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): finished with status 'done'
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.56.0.post1-py3-none-any.whl.metadata (751 bytes)
2024-07-03T14:35:41: Collecting azureml-inference-server-http~=1.2.1 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_inference_server_http-1.2.2-py3-none-any.whl.metadata (9.8 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.56.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.56.0-py3-none-any.whl.metadata (3.1 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.56.0 (from azureml-dataset-runtime[fuse]~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.56.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting numpy>=1.16.0 (from pymanopt==2.2.1.dev14+g1de3b6f->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 2))
2024-07-03T14:35:41:   Downloading numpy-2.0.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 60.9/60.9 kB 3.2 MB/s eta 0:00:00
2024-07-03T14:35:41: Collecting scipy!=1.10.*,!=1.11.*,!=1.12.*,>=1.0 (from pymanopt==2.2.1.dev14+g1de3b6f->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 2))
2024-07-03T14:35:41:   Downloading scipy-1.14.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 60.8/60.8 kB 4.3 MB/s eta 0:00:00
2024-07-03T14:35:41: Collecting category-encoders<3,>=2.6.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading category_encoders-2.6.3-py2.py3-none-any.whl.metadata (8.0 kB)
2024-07-03T14:35:41: Collecting cornac<3,>=1.15.2 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading cornac-2.2.1-cp312-cp312-manylinux1_x86_64.whl.metadata (30 kB)
2024-07-03T14:35:41: Collecting hyperopt<1,>=0.2.7 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading hyperopt-0.2.7-py2.py3-none-any.whl.metadata (1.7 kB)
2024-07-03T14:35:41: Collecting lightgbm<5,>=4.0.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading lightgbm-4.4.0-py3-none-manylinux_2_28_x86_64.whl.metadata (19 kB)
2024-07-03T14:35:41: Collecting locust<3,>=2.12.2 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading locust-2.29.1-py3-none-any.whl.metadata (7.4 kB)
2024-07-03T14:35:41: Collecting memory-profiler<1,>=0.61.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading memory_profiler-0.61.0-py3-none-any.whl.metadata (20 kB)
2024-07-03T14:35:41: Collecting nltk<4,>=3.8.1 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading nltk-3.8.1-py3-none-any.whl.metadata (2.8 kB)
2024-07-03T14:35:41: Collecting notebook<8,>=6.5.5 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading notebook-7.2.1-py3-none-any.whl.metadata (10 kB)
2024-07-03T14:35:41: Collecting numba<1,>=0.57.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading numba-0.60.0-cp312-cp312-manylinux2014_x86_64.manylinux_2_17_x86_64.whl.metadata (2.7 kB)
2024-07-03T14:35:41: Collecting pandas<3.0.0,>2.0.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading pandas-2.2.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (19 kB)
2024-07-03T14:35:41: Collecting retrying<2,>=1.3.4 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading retrying-1.3.4-py3-none-any.whl.metadata (6.9 kB)
2024-07-03T14:35:41: Collecting scikit-learn<2,>=1.2.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading scikit_learn-1.5.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (12 kB)
2024-07-03T14:35:41: Collecting scikit-surprise>=1.1.3 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading scikit_surprise-1.1.4.tar.gz (154 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 154.4/154.4 kB 13.5 MB/s eta 0:00:00
2024-07-03T14:35:41:   Installing build dependencies: started
2024-07-03T14:35:41:   Installing build dependencies: finished with status 'done'
2024-07-03T14:35:41:   Getting requirements to build wheel: started
2024-07-03T14:35:41:   Getting requirements to build wheel: finished with status 'done'
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): started
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): finished with status 'done'
2024-07-03T14:35:41: Collecting seaborn<1,>=0.13.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading seaborn-0.13.2-py3-none-any.whl.metadata (5.4 kB)
2024-07-03T14:35:41: Collecting transformers<5,>=4.27.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading transformers-4.42.3-py3-none-any.whl.metadata (43 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 43.6/43.6 kB 2.8 MB/s eta 0:00:00
2024-07-03T14:35:41: Collecting pandera>=0.15.0 (from pandera[strategies]>=0.15.0; python_version >= "3.9"->recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading pandera-0.20.1-py3-none-any.whl.metadata (15 kB)
2024-07-03T14:35:41: Collecting black>=23.3.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading black-24.4.2-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (77 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 77.1/77.1 kB 6.0 MB/s eta 0:00:00
2024-07-03T14:35:41: Collecting pytest>=7.2.1 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading pytest-8.2.2-py3-none-any.whl.metadata (7.6 kB)
2024-07-03T14:35:41: Collecting pytest-cov>=4.1.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading pytest_cov-5.0.0-py3-none-any.whl.metadata (27 kB)
2024-07-03T14:35:41: Collecting pytest-mock>=3.10.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading pytest_mock-3.14.0-py3-none-any.whl.metadata (3.8 kB)
2024-07-03T14:35:41: Collecting pytz (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pytz-2024.1-py2.py3-none-any.whl.metadata (22 kB)
2024-07-03T14:35:41: Collecting backports.tempfile (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading backports.tempfile-1.0-py2.py3-none-any.whl.metadata (2.3 kB)
2024-07-03T14:35:41: Collecting pathspec<1.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pathspec-0.12.1-py3-none-any.whl.metadata (21 kB)
2024-07-03T14:35:41: Collecting requests<3.0.0,>=2.19.1 (from requests[socks]<3.0.0,>=2.19.1->azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading requests-2.32.3-py3-none-any.whl.metadata (4.6 kB)
2024-07-03T14:35:41: Collecting msal<2.0.0,>=1.15.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading msal-1.29.0-py3-none-any.whl.metadata (11 kB)
2024-07-03T14:35:41: Collecting msal-extensions<=2.0.0,>=0.3.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading msal_extensions-1.2.0-py3-none-any.whl.metadata (7.6 kB)
2024-07-03T14:35:41: Collecting knack<0.12.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading knack-0.11.0-py3-none-any.whl.metadata (5.2 kB)
2024-07-03T14:35:41: Collecting azure-core<2.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_core-1.30.2-py3-none-any.whl.metadata (37 kB)
2024-07-03T14:35:41: Collecting pkginfo (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pkginfo-1.11.1-py3-none-any.whl.metadata (11 kB)
2024-07-03T14:35:41: Collecting argcomplete<4 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading argcomplete-3.4.0-py3-none-any.whl.metadata (16 kB)
2024-07-03T14:35:41: Collecting humanfriendly<11.0,>=4.7 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading humanfriendly-10.0-py2.py3-none-any.whl.metadata (9.2 kB)
2024-07-03T14:35:41: Collecting paramiko<4.0.0,>=2.0.8 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading paramiko-3.4.0-py3-none-any.whl.metadata (4.4 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-resource<=24.0.0,>=15.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_resource-23.1.1-py3-none-any.whl.metadata (37 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-containerregistry<11,>=8.2.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_containerregistry-10.3.0-py3-none-any.whl.metadata (23 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-storage<=22.0.0,>=16.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_storage-21.2.1-py3-none-any.whl.metadata (31 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-keyvault<11.0.0,>=0.40.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_keyvault-10.3.0-py3-none-any.whl.metadata (15 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-authorization<5,>=0.40.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_authorization-4.0.0-py3-none-any.whl.metadata (18 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-network<=26.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_network-25.4.0-py3-none-any.whl.metadata (83 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 83.7/83.7 kB 7.9 MB/s eta 0:00:00
2024-07-03T14:35:41: Collecting azure-graphrbac<1.0.0,>=0.40.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_graphrbac-0.61.1-py2.py3-none-any.whl.metadata (10 kB)
2024-07-03T14:35:41: Collecting azure-common<2.0.0,>=1.1.12 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_common-1.1.28-py2.py3-none-any.whl.metadata (5.0 kB)
2024-07-03T14:35:41: Collecting msrest<=0.7.1,>=0.5.1 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading msrest-0.7.1-py3-none-any.whl.metadata (21 kB)
2024-07-03T14:35:41: Collecting msrestazure<=0.6.4,>=0.4.33 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading msrestazure-0.6.4-py2.py3-none-any.whl.metadata (15 kB)
2024-07-03T14:35:41: Collecting urllib3<3.0.0,>1.26.17 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading urllib3-2.2.2-py3-none-any.whl.metadata (6.4 kB)
2024-07-03T14:35:41: Collecting packaging<=25.0,>=20.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Using cached packaging-24.1-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting python-dateutil<3.0.0,>=2.7.3 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
2024-07-03T14:35:41: Collecting ndg-httpsclient<=0.5.1 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading ndg_httpsclient-0.5.1-py3-none-any.whl.metadata (6.2 kB)
2024-07-03T14:35:41: Collecting SecretStorage<4.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading SecretStorage-3.3.3-py3-none-any.whl.metadata (4.0 kB)
2024-07-03T14:35:41: Collecting jsonpickle<4.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading jsonpickle-3.2.2-py3-none-any.whl.metadata (7.2 kB)
2024-07-03T14:35:41: Collecting contextlib2<22.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading contextlib2-21.6.0-py2.py3-none-any.whl.metadata (4.1 kB)
2024-07-03T14:35:41: Collecting docker<8.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading docker-7.1.0-py3-none-any.whl.metadata (3.8 kB)
2024-07-03T14:35:41: Collecting PyJWT<3.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading PyJWT-2.8.0-py3-none-any.whl.metadata (4.2 kB)
2024-07-03T14:35:41: Collecting adal<=1.2.7,>=1.2.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading adal-1.2.7-py2.py3-none-any.whl.metadata (6.9 kB)
2024-07-03T14:35:41: Collecting pyopenssl<25.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pyOpenSSL-24.1.0-py3-none-any.whl.metadata (12 kB)
2024-07-03T14:35:41: Collecting jmespath<2.0.0 (from azureml-core~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading jmespath-1.0.1-py3-none-any.whl.metadata (7.6 kB)
2024-07-03T14:35:41: Collecting azureml-dataprep<5.2.0a,>=5.1.0a (from azureml-dataset-runtime~=1.56.0->azureml-dataset-runtime[fuse]~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataprep-5.1.6-py3-none-any.whl.metadata (2.2 kB)
2024-07-03T14:35:41: Collecting pyarrow>=0.17.0 (from azureml-dataset-runtime~=1.56.0->azureml-dataset-runtime[fuse]~=1.56.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pyarrow-16.1.0-cp312-cp312-manylinux_2_28_x86_64.whl.metadata (3.0 kB)
2024-07-03T14:35:41: INFO: pip is looking at multiple versions of azureml-dataset-runtime to determine which version is compatible with other requirements. This could take a while.
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.56.0-py3-none-any.whl.metadata (745 bytes)
2024-07-03T14:35:41: Collecting azureml-inference-server-http~=1.0.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_inference_server_http-1.0.0-py3-none-any.whl.metadata (9.0 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.55.0-py3-none-any.whl.metadata (745 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.55.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.55.0.post2-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.55.0 (from azureml-dataset-runtime[fuse]~=1.55.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.55.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.54.0-py3-none-any.whl.metadata (753 bytes)
2024-07-03T14:35:41: Collecting azureml-inference-server-http~=0.8.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_inference_server_http-0.8.4.2-py3-none-any.whl.metadata (12 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.54.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.54.0.post1-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.54.0 (from azureml-dataset-runtime[fuse]~=1.54.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.54.0.post1-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataprep<4.14.0a,>=4.12.0a (from azureml-dataset-runtime~=1.54.0->azureml-dataset-runtime[fuse]~=1.54.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataprep-4.12.10-py3-none-any.whl.metadata (2.1 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.54.0 (from azureml-dataset-runtime[fuse]~=1.54.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.54.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting pyarrow<=11.0.0,>=0.17.0 (from azureml-dataset-runtime~=1.54.0->azureml-dataset-runtime[fuse]~=1.54.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pyarrow-11.0.0.tar.gz (1.0 MB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.0/1.0 MB 48.3 MB/s eta 0:00:00
2024-07-03T14:35:41:   Installing build dependencies: started
2024-07-03T14:35:41:   Installing build dependencies: finished with status 'done'
2024-07-03T14:35:41:   Getting requirements to build wheel: started
2024-07-03T14:35:41:   Getting requirements to build wheel: finished with status 'done'
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): started
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): finished with status 'done'
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.53.0-py3-none-any.whl.metadata (753 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.53.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.53.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.53.0 (from azureml-dataset-runtime[fuse]~=1.53.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.53.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.52.0-py3-none-any.whl.metadata (753 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.52.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.52.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.52.0 (from azureml-dataset-runtime[fuse]~=1.52.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.52.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataprep<4.12.0a,>=4.11.3a (from azureml-dataset-runtime~=1.52.0->azureml-dataset-runtime[fuse]~=1.52.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataprep-4.11.7-py3-none-any.whl.metadata (2.1 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.51.0-py3-none-any.whl.metadata (753 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.51.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.51.0.post1-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.51.0 (from azureml-dataset-runtime[fuse]~=1.51.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.51.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataprep<4.11.0a,>=4.10.0a (from azureml-dataset-runtime~=1.51.0->azureml-dataset-runtime[fuse]~=1.51.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataprep-4.10.9-py3-none-any.whl.metadata (2.1 kB)
2024-07-03T14:35:41: Collecting pyarrow<=9.0.0,>=0.17.0 (from azureml-dataset-runtime~=1.51.0->azureml-dataset-runtime[fuse]~=1.51.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pyarrow-9.0.0.tar.gz (873 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 873.1/873.1 kB 34.1 MB/s eta 0:00:00
2024-07-03T14:35:41:   Installing build dependencies: started
2024-07-03T14:35:41:   Installing build dependencies: finished with status 'done'
2024-07-03T14:35:41:   Getting requirements to build wheel: started
2024-07-03T14:35:41:   Getting requirements to build wheel: finished with status 'done'
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): started
2024-07-03T14:35:41:   Preparing metadata (pyproject.toml): finished with status 'done'
2024-07-03T14:35:41: INFO: pip is still looking at multiple versions of azureml-dataset-runtime to determine which version is compatible with other requirements. This could take a while.
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.50.0-py3-none-any.whl.metadata (753 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.50.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.50.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.50.0 (from azureml-dataset-runtime[fuse]~=1.50.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.50.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.49.0-py3-none-any.whl.metadata (753 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.49.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.49.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.49.0 (from azureml-dataset-runtime[fuse]~=1.49.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.49.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataprep<4.10.0a,>=4.9.0a (from azureml-dataset-runtime~=1.49.0->azureml-dataset-runtime[fuse]~=1.49.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataprep-4.9.7-py3-none-any.whl.metadata (2.1 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.48.0-py3-none-any.whl.metadata (753 bytes)
2024-07-03T14:35:41: Collecting azureml-inference-server-http~=0.7.2 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_inference_server_http-0.7.7-py3-none-any.whl.metadata (9.5 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.48.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.48.0-py3-none-any.whl.metadata (3.1 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.48.0 (from azureml-dataset-runtime[fuse]~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.48.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataprep<4.9.0a,>=4.8.0a (from azureml-dataset-runtime~=1.48.0->azureml-dataset-runtime[fuse]~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataprep-4.8.6-py3-none-any.whl.metadata (2.1 kB)
2024-07-03T14:35:41: Collecting msal-extensions<=1.0.0,>=0.3.0 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading msal_extensions-1.0.0-py2.py3-none-any.whl.metadata (7.3 kB)
2024-07-03T14:35:41: Collecting knack~=0.10.0 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading knack-0.10.1-py3-none-any.whl.metadata (5.2 kB)
2024-07-03T14:35:41: Collecting argcomplete<3 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading argcomplete-2.1.2-py3-none-any.whl.metadata (17 kB)
2024-07-03T14:35:41: Collecting paramiko<3.0.0,>=2.0.8 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading paramiko-2.12.0-py2.py3-none-any.whl.metadata (4.6 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-resource<22.0.0,>=15.0.0 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_resource-21.2.1-py3-none-any.whl.metadata (29 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-storage<21.0.0,>=16.0.0 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_storage-20.1.0-py3-none-any.whl.metadata (27 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-authorization<4,>=0.40.0 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_authorization-3.0.0-py3-none-any.whl.metadata (14 kB)
2024-07-03T14:35:41: Collecting urllib3<2.0.0,>=1.23 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading urllib3-1.26.19-py2.py3-none-any.whl.metadata (49 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 49.3/49.3 kB 2.9 MB/s eta 0:00:00
2024-07-03T14:35:41: Collecting packaging<22.0,>=20.0 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading packaging-21.3-py3-none-any.whl.metadata (15 kB)
2024-07-03T14:35:41: Collecting cryptography!=1.9,!=2.0.*,!=2.1.*,!=2.2.*,<41 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading cryptography-40.0.2-cp36-abi3-manylinux_2_28_x86_64.whl.metadata (5.4 kB)
2024-07-03T14:35:41: Collecting jsonpickle<3.0.0 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading jsonpickle-2.2.0-py2.py3-none-any.whl.metadata (7.6 kB)
2024-07-03T14:35:41: Collecting docker<7.0.0 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading docker-6.1.3-py3-none-any.whl.metadata (3.5 kB)
2024-07-03T14:35:41: Collecting pyopenssl<23.0.0 (from azureml-core~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pyOpenSSL-22.1.0-py3-none-any.whl.metadata (8.9 kB)
2024-07-03T14:35:41: Collecting fusepy<4.0.0,>=3.0.1 (from azureml-dataset-runtime[fuse]~=1.48.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading fusepy-3.0.1.tar.gz (11 kB)
2024-07-03T14:35:41:   Preparing metadata (setup.py): started
2024-07-03T14:35:41:   Preparing metadata (setup.py): finished with status 'done'
2024-07-03T14:35:41: Collecting opencensus-ext-azure~=1.1.0 (from azureml-inference-server-http~=0.7.2->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading opencensus_ext_azure-1.1.13-py2.py3-none-any.whl.metadata (16 kB)
2024-07-03T14:35:41: Collecting flask-cors~=3.0.1 (from azureml-inference-server-http~=0.7.2->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading Flask_Cors-3.0.10-py2.py3-none-any.whl.metadata (5.4 kB)
2024-07-03T14:35:41: Collecting gunicorn==20.1.0 (from azureml-inference-server-http~=0.7.2->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading gunicorn-20.1.0-py3-none-any.whl.metadata (3.8 kB)
2024-07-03T14:35:41: INFO: pip is looking at multiple versions of azureml-inference-server-http to determine which version is compatible with other requirements. This could take a wh[91m
2024-07-03T14:35:41: CondaEnvException: Pip failed

2024-07-03T14:35:41: [0mile.
2024-07-03T14:35:41: Collecting azureml-inference-server-http~=0.7.2 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_inference_server_http-0.7.6-py3-none-any.whl.metadata (9.3 kB)
2024-07-03T14:35:41: Collecting flask<2.2.0 (from azureml-inference-server-http~=0.7.2->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading Flask-2.1.3-py3-none-any.whl.metadata (3.9 kB)
2024-07-03T14:35:41: Collecting inference-schema~=1.4.0 (from azureml-inference-server-http~=0.7.2->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading inference_schema-1.4.1-py3-none-any.whl.metadata (2.4 kB)
2024-07-03T14:35:41: Requirement already satisfied: setuptools>=3.0 in /azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc/lib/python3.12/site-packages (from gunicorn==20.1.0->azureml-inference-server-http~=0.7.2->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1)) (68.0.0)
2024-07-03T14:35:41: Collecting click>=8.0.0 (from black>=23.3.0->recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading click-8.1.7-py3-none-any.whl.metadata (3.0 kB)
2024-07-03T14:35:41: Collecting mypy-extensions>=0.4.3 (from black>=23.3.0->recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading mypy_extensions-1.0.0-py3-none-any.whl.metadata (1.1 kB)
2024-07-03T14:35:41: INFO: pip is looking at multiple versions of black to determine which version is compatible with other requirements. This could take a while.
2024-07-03T14:35:41: Collecting black>=23.3.0 (from recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading black-24.4.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (76 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 76.8/76.8 kB 3.1 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-24.4.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (76 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 76.4/76.4 kB 2.2 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-24.3.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (75 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 76.0/76.0 kB 6.0 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-24.2.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (74 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 74.6/74.6 kB 6.0 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-24.1.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (73 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 73.3/73.3 kB 6.7 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-24.1.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (72 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 72.9/72.9 kB 5.5 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-23.12.1-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (68 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 69.0/69.0 kB 5.8 MB/s eta 0:00:00
2024-07-03T14:35:41: INFO: pip is still looking at multiple versions of black to determine which version is compatible with other requirements. This could take a while.
2024-07-03T14:35:41:   Downloading black-23.12.0-cp312-cp312-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (68 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 68.9/68.9 kB 6.2 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-23.11.0-py3-none-any.whl.metadata (66 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 66.9/66.9 kB 5.3 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-23.10.1-py3-none-any.whl.metadata (66 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 66.0/66.0 kB 5.6 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-23.10.0-py3-none-any.whl.metadata (66 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 66.2/66.2 kB 4.7 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-23.9.1-py3-none-any.whl.metadata (65 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 65.1/65.1 kB 4.7 MB/s eta 0:00:00
2024-07-03T14:35:41: INFO: This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. See https://pip.pypa.io/warnings/backtracking for guidance. If you want to abort this run, press Ctrl + C.
2024-07-03T14:35:41:   Downloading black-23.9.0-py3-none-any.whl.metadata (64 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 64.5/64.5 kB 5.3 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-23.7.0-py3-none-any.whl.metadata (63 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 63.4/63.4 kB 2.7 MB/s eta 0:00:00
2024-07-03T14:35:41:   Downloading black-23.3.0-py3-none-any.whl.metadata (61 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.0/61.0 kB 4.3 MB/s eta 0:00:00
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.47.0-py3-none-any.whl.metadata (830 bytes)
2024-07-03T14:35:41: Collecting platformdirs>=2 (from black>=23.3.0->recommenders[dev]@ git+https://github.com/recommenders-team/recommenders.git@842308827aa02d25c3863f2f6552351a94eb19d0->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 3))
2024-07-03T14:35:41:   Downloading platformdirs-4.2.2-py3-none-any.whl.metadata (11 kB)
2024-07-03T14:35:41: Collecting configparser==3.7.4 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading configparser-3.7.4-py2.py3-none-any.whl.metadata (11 kB)
2024-07-03T14:35:41: Collecting json-logging-py==0.2 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading json-logging-py-0.2.tar.gz (3.6 kB)
2024-07-03T14:35:41:   Preparing metadata (setup.py): started
2024-07-03T14:35:41:   Preparing metadata (setup.py): finished with status 'done'
2024-07-03T14:35:41: Collecting azureml-core~=1.47.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.47.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.47.0 (from azureml-dataset-runtime[fuse]~=1.47.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.47.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-authorization<3,>=0.40.0 (from azureml-core~=1.47.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_authorization-2.0.0-py2.py3-none-any.whl.metadata (11 kB)
2024-07-03T14:35:41: INFO: pip is looking at multiple versions of azureml-core to determine which version is compatible with other requirements. This could take a while.
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.46.0-py3-none-any.whl.metadata (830 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.46.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.46.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.46.0 (from azureml-dataset-runtime[fuse]~=1.46.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.46.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting knack~=0.9.0 (from azureml-core~=1.46.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading knack-0.9.0-py3-none-any.whl.metadata (5.2 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-storage<=20.0.0,>=16.0.0 (from azureml-core~=1.46.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_storage-20.0.0-py3-none-any.whl.metadata (27 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.45.0-py3-none-any.whl.metadata (830 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.45.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.45.0.post3-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.45.0 (from azureml-dataset-runtime[fuse]~=1.45.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.45.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.45.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.45.0.post2-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41:   Downloading azureml_core-1.45.0.post1-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41:   Downloading azureml_core-1.45.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.44.0-py3-none-any.whl.metadata (830 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.44.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.44.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.44.0 (from azureml-dataset-runtime[fuse]~=1.44.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.44.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting urllib3<=1.26.9,>=1.23 (from azureml-core~=1.44.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading urllib3-1.26.9-py2.py3-none-any.whl.metadata (46 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 46.3/46.3 kB 3.5 MB/s eta 0:00:00
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.43.0-py3-none-any.whl.metadata (830 bytes)
2024-07-03T14:35:41: Collecting azureml-inference-server-http~=0.4.1 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_inference_server_http-0.4.14-py3-none-any.whl.metadata (2.1 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.43.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.43.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.43.0 (from azureml-dataset-runtime[fuse]~=1.43.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.43.0.post2-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-core<=1.3.0 (from azureml-core~=1.43.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_core-1.3.0-py2.py3-none-any.whl.metadata (3.8 kB)
2024-07-03T14:35:41: Collecting msrest<0.7.0,>=0.5.1 (from azureml-core~=1.43.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading msrest-0.6.21-py2.py3-none-any.whl.metadata (21 kB)
2024-07-03T14:35:41: INFO: pip is still looking at multiple versions of azureml-core to determine which version is compatible with other requirements. This could take a while.
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.42.0-py3-none-any.whl.metadata (830 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.42.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.42.0.post1-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.42.0 (from azureml-dataset-runtime[fuse]~=1.42.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.42.0-py3-none-any.whl.metadata (1.2 kB)
2024-07-03T14:35:41: Collecting azure-core<=1.22.1 (from azureml-core~=1.42.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_core-1.22.1-py3-none-any.whl.metadata (31 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-resource<=21.0.0,>=15.0.0 (from azureml-core~=1.42.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_resource-21.0.0-py3-none-any.whl.metadata (29 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-containerregistry<10,>=8.2.0 (from azureml-core~=1.42.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_containerregistry-9.1.0-py3-none-any.whl.metadata (21 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-keyvault<10.0.0,>=0.40.0 (from azureml-core~=1.42.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_keyvault-9.3.0-py2.py3-none-any.whl.metadata (12 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.42.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.42.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.41.0-py3-none-any.whl.metadata (848 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.41.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.41.0.post3-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.41.0 (from azureml-dataset-runtime[fuse]~=1.41.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.41.0-py3-none-any.whl.metadata (1.3 kB)
2024-07-03T14:35:41: Collecting msal-extensions<0.4,>=0.3.0 (from azureml-core~=1.41.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading msal_extensions-0.3.1-py2.py3-none-any.whl.metadata (6.3 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-resource<21.0.0,>=15.0.0 (from azureml-core~=1.41.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_resource-20.1.0-py3-none-any.whl.metadata (28 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-storage<20.0.0,>=16.0.0 (from azureml-core~=1.41.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_storage-19.1.0-py3-none-any.whl.metadata (26 kB)
2024-07-03T14:35:41: Collecting urllib3<=1.26.7,>=1.23 (from azureml-core~=1.41.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading urllib3-1.26.7-py2.py3-none-any.whl.metadata (44 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 44.5/44.5 kB 156.8 kB/s eta 0:00:00
2024-07-03T14:35:41: Collecting azureml-core~=1.41.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.41.0.post2-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41:   Downloading azureml_core-1.41.0.post1-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: INFO: This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. See https://pip.pypa.io/warnings/backtracking for guidance. If you want to abort this run, press Ctrl + C.
2024-07-03T14:35:41:   Downloading azureml_core-1.41.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.40.0-py3-none-any.whl.metadata (848 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.40.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.40.0.post2-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.40.0 (from azureml-dataset-runtime[fuse]~=1.40.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.40.0-py3-none-any.whl.metadata (1.3 kB)
2024-07-03T14:35:41: Collecting azure-core<1.22 (from azureml-core~=1.40.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_core-1.21.1-py2.py3-none-any.whl.metadata (30 kB)
2024-07-03T14:35:41: Collecting argcomplete<2.0 (from azureml-core~=1.40.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading argcomplete-1.12.3-py2.py3-none-any.whl.metadata (18 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-containerregistry<9.0.0,>=8.2.0 (from azureml-core~=1.40.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_containerregistry-8.2.0-py2.py3-none-any.whl.metadata (20 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-authorization<1.0.0,>=0.40.0 (from azureml-core~=1.40.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_authorization-0.61.0-py2.py3-none-any.whl.metadata (5.1 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.40.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.40.0.post1-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41:   Downloading azureml_core-1.40.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.39.0-py3-none-any.whl.metadata (848 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.39.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.39.0.post1-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.39.0 (from azureml-dataset-runtime[fuse]~=1.39.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.39.0-py3-none-any.whl.metadata (1.3 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.39.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.39.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.38.0-py3-none-any.whl.metadata (848 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.38.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.38.0.post2-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.38.0 (from azureml-dataset-runtime[fuse]~=1.38.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.38.0-py3-none-any.whl.metadata (1.3 kB)
2024-07-03T14:35:41: Collecting knack~=0.8.2 (from azureml-core~=1.38.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading knack-0.8.2-py3-none-any.whl.metadata (5.2 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.38.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.38.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.37.0-py3-none-any.whl.metadata (848 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.37.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.37.0.post2-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.37.0 (from azureml-dataset-runtime[fuse]~=1.37.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.37.0-py3-none-any.whl.metadata (1.3 kB)
2024-07-03T14:35:41: Collecting azure-core<1.21 (from azureml-core~=1.37.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_core-1.20.1-py2.py3-none-any.whl.metadata (30 kB)
2024-07-03T14:35:41: Collecting humanfriendly<10.0,>=4.7 (from azureml-core~=1.37.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading humanfriendly-9.2-py2.py3-none-any.whl.metadata (8.8 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-resource<20.0.0,>=15.0.0 (from azureml-core~=1.37.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_resource-19.0.0-py2.py3-none-any.whl.metadata (27 kB)
2024-07-03T14:35:41: Collecting azureml-core~=1.37.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.37.0.post1-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41:   Downloading azureml_core-1.37.0-py3-none-any.whl.metadata (3.2 kB)
2024-07-03T14:35:41: Collecting azureml-defaults (from -r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_defaults-1.36.0-py3-none-any.whl.metadata (848 bytes)
2024-07-03T14:35:41: Collecting azureml-core~=1.36.0 (from azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_core-1.36.0.post2-py3-none-any.whl.metadata (2.9 kB)
2024-07-03T14:35:41: Collecting azureml-dataset-runtime~=1.36.0 (from azureml-dataset-runtime[fuse]~=1.36.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataset_runtime-1.36.0-py3-none-any.whl.metadata (1.3 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-resource<15.0.0,>=1.2.1 (from azureml-core~=1.36.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_resource-13.0.0-py2.py3-none-any.whl.metadata (24 kB)
2024-07-03T14:35:41: Collecting azure-mgmt-storage<16.0.0,>=1.5.0 (from azureml-core~=1.36.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azure_mgmt_storage-11.2.0-py2.py3-none-any.whl.metadata (17 kB)
2024-07-03T14:35:41: Collecting cryptography!=1.9,!=2.0.*,!=2.1.*,!=2.2.*,<4.0.0 (from azureml-core~=1.36.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading cryptography-3.4.8-cp36-abi3-manylinux_2_24_x86_64.whl.metadata (5.2 kB)
2024-07-03T14:35:41: Collecting docker<6.0.0 (from azureml-core~=1.36.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading docker-5.0.3-py2.py3-none-any.whl.metadata (3.5 kB)
2024-07-03T14:35:41: Collecting pyopenssl<21.0.0 (from azureml-core~=1.36.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pyOpenSSL-20.0.1-py2.py3-none-any.whl.metadata (6.7 kB)
2024-07-03T14:35:41: Collecting jmespath<1.0.0 (from azureml-core~=1.36.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading jmespath-0.10.0-py2.py3-none-any.whl.metadata (8.0 kB)
2024-07-03T14:35:41: Collecting azureml-dataprep<2.25.0a,>=2.24.0a (from azureml-dataset-runtime~=1.36.0->azureml-dataset-runtime[fuse]~=1.36.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading azureml_dataprep-2.24.4-py3-none-any.whl.metadata (2.0 kB)
2024-07-03T14:35:41: Collecting pyarrow<4.0.0,>=0.17.0 (from azureml-dataset-runtime~=1.36.0->azureml-dataset-runtime[fuse]~=1.36.0->azureml-defaults->-r /azureml-environment-setup/condaenv.bfz0wkpt.requirements.txt (line 1))
2024-07-03T14:35:41:   Downloading pyarrow-3.0.0.tar.gz (682 kB)
2024-07-03T14:35:41:      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 682.2/682.2 kB 40.1 MB/s eta 0:00:00
2024-07-03T14:35:41:   Installing build dependencies: started
2024-07-03T14:35:41:   Installing build dependencies: finished with status 'error'

2024-07-03T14:35:41: failed
2024-07-03T14:35:42: The command '/bin/sh -c ldconfig /usr/local/cuda/lib64/stubs && conda env create -p /azureml-envs/azureml_9eab27162e8e2d6f98a42f938c8ebacc -f azureml-environment-setup/mutated_conda_dependencies.yml && rm -rf "$HOME/.cache/pip" && conda clean -aqy && CONDA_ROOT_DIR=$(conda info --root) && rm -rf "$CONDA_ROOT_DIR/pkgs" && find "$CONDA_ROOT_DIR" -type d -name __pycache__ -exec rm -rf {} + && ldconfig' returned a non-zero code: 1


2024-07-03T14:35:42: CalledProcessError(1, ['docker', 'build', '-f', 'azureml-environment-setup/Dockerfile', '.', '-t', '978a92daa2ad4447aae1b21196dd4a9b.azurecr.io/azureml/azureml_fa83f0efe189b6dbcbb6f109441a392a', '-t', '978a92daa2ad4447aae1b21196dd4a9b.azurecr.io/azureml/azureml_fa83f0efe189b6dbcbb6f109441a392a:1'])

2024-07-03T14:35:42: Building docker image failed with exit code: 1

2024-07-03T14:35:42: Logging out of Docker registry: 978a92daa2ad4447aae1b21196dd4a9b.azurecr.io
2024-07-03T14:35:42: Removing login credentials for https://index.docker.io/v1/


2024-07-03T14:35:42: Cleaning up all outstanding Run operations, waiting 300.0 seconds
2024-07-03T14:35:42: 0 items cleaning up...
2024-07-03T14:35:42: Cleanup took 1.1920928955078125e-06 seconds
2024-07-03T14:35:42: Traceback (most recent call last):
  File "/mnt/azureml/cr/j/11fd4e292a9e487b97d3822cd5bbc334/exe/wd/docker_utilities.py", line 152, in _docker_build_or_error
    docker_execute_function(docker_command, build_command, print_command_args=True)
  File "/mnt/azureml/cr/j/11fd4e292a9e487b97d3822cd5bbc334/exe/wd/docker_utilities.py", line 23, in docker_execute_function
    return killable_subprocess.check_call(command_args, *popen_args,
  File "/mnt/azureml/cr/j/11fd4e292a9e487b97d3822cd5bbc334/exe/wd/killable_subprocess.py", line 261, in check_call
    raise subprocess.CalledProcessError(process.returncode, cmd)
subprocess.CalledProcessError: Command '['docker', 'build', '-f', 'azureml-environment-setup/Dockerfile', '.', '-t', '978a92daa2ad4447aae1b21196dd4a9b.azurecr.io/azureml/azureml_fa83f0efe189b6dbcbb6f109441a392a', '-t', '978a92daa2ad4447aae1b21196dd4a9b.azurecr.io/azureml/azureml_fa83f0efe189b6dbcbb6f109441a392a:1']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "script.py", line 108, in <module>
    docker_utilities._docker_build_or_error(
  File "/mnt/azureml/cr/j/11fd4e292a9e487b97d3822cd5bbc334/exe/wd/docker_utilities.py", line 156, in _docker_build_or_error
    _write_error_and_exit(error_msg, error_file_path=error_file_path)
  File "/mnt/azureml/cr/j/11fd4e292a9e487b97d3822cd5bbc334/exe/wd/docker_utilities.py", line 217, in _write_error_and_exit
    sys.exit(1)
SystemExit: 1
```

FYI @daviddavo

Reply:
@SimonYansenZhao I think it would be useful while you are doing the migration to azureml v2 to output the logs of the VM if the test fails, otherwise we are a little bit blind

Reply:
@miguelgfierro I can try to put logs into GitHub Actions artifact, so that anyone can see logs.